tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -7,7 +7,12 @@
 import boto3
 from typing import Dict, Any, Optional, List
 from datetime import datetime, timezone
+import os
+import boto3
```

Comment:
boto3 imported twice. Once in line 7 and then again here.

Reply:
removed one